lib/config/fileconf.go

```diff
@@ -91,7 +93,10 @@ type FileConfig struct {
 func ReadFromFile(filePath string) (*FileConfig, error) {
 	f, err := os.Open(filePath)
 	if err != nil {
-		return nil, trace.Wrap(err, fmt.Sprintf("failed to open file: %v", filePath))
+		if errors.Is(err, fs.ErrPermission) {
+			return nil, trace.Wrap(err, fmt.Sprintf("failed to open file for Teleport configuration: %v. Ensure that you are running as a user with appropriate permissions.", filePath))
```

Comment:
```suggestion
			return nil, trace.Wrap(err, "failed to open file for Teleport configuration: %v. Ensure that you are running as a user with appropriate permissions.", filePath)
```

No need for the `Sprintf`. (same comment applies below as well)